container/helm/README.md

```diff
@@ -1,46 +1,35 @@
-### Helm Chart for openQA inside Kubernetes
+# Helm Chart for openQA inside Kubernetes
 
 Prerequisites:
-1. A Kubernetes cluster (for example _k3s_ will do fine)
-2. Installed and configured Helm
-
-For more information please consult corresponding documentation for [k3s](https://rancher.com/docs/k3s/latest/en/) or [Helm](https://helm.sh/docs).
```

Comment:
Why remove the helpful pointers to k3s?

Reply:
I thought it'd be better if it didn't mention specific technologies, but be more general instead, as the charts are independent of the cluster implementation. But we can keep it.

Reply:
btw., k3s is mentioned not as a randomly chosen technology but as "our" technology (or our product if you wish) so in my opinion the "for example k3s will do fine" statement should be read in the same way as saying "you can install openQA on Tumbleweed"

Reply:
I see, that makes sense. I'll add a mention of *k3s* back in.